addons/auth/package.json

```diff
@@ -46,11 +46,11 @@
     "ember-disable-prototype-extensions": "^1.1.3",
     "ember-export-application-global": "^2.0.1",
     "ember-load-initializers": "^2.1.2",
-    "ember-maybe-import-regenerator": "^1.0.0",
     "ember-page-title": "^7.0.0",
     "ember-qunit": "^6.0.0",
     "ember-resolver": "^8.0.3",
-    "ember-source": "~3.28.8",
+    "ember-source": "~4.4.0",
+    "ember-source-channel-url": "^3.0.0",
```

Comment:
I add `ember-source-channel-url` as a devDependency.

The cause is the linter complaining about [this rule](https://github.com/mysticatea/eslint-plugin-node/blob/master/docs/rules/no-extraneous-require.md).

It is used JUST by `ember-try`, so an alternative to this will be ignoring the especific rule mentioned above for `config/ember-try`.

I already add to `.npmignore` the `config/ember-try` but still complaining about the issue mentioned above. Happy to discuss other solutions.

Reply:
I didn't have any issues locally.  I pushed up an updated yarn.lock so perhaps that will help.

Reply:
Yes, I validate your comment. Pull latest changes, re-install deps, run the addon and everything seems fine to me.